lint-js.sh

```diff
@@ -1,34 +1,11 @@
 #!/bin/bash
 
-while getopts 'c:p:n:t:f:' flag; do
-	case "${flag}" in
-		c) cwdiswppslinter=${OPTARG} ;;
-		p) plugindirname=${OPTARG} ;;
-		n) namespace=${OPTARG} ;;
-		t) textdomain=${OPTARG} ;;
-		f) fix=${OPTARG} ;;
-	esac
-done
-
-# Get the absolute path to the plugin we want to check.
-if [ "$cwdiswppslinter" = "1" ]; then
-	plugindir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")" )" )" )" )/$plugindirname"
-	wpcontentdir="./../../../../"
-	scriptsdir="$(dirname "$(realpath "$0")" )/"
-else
-	cwdiswppslinter=0
-	plugindir="$(dirname "$(dirname "$(realpath "$0")" )" )"
-	wpcontentdir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath $0)" )" )" )" )"
-	scriptsdir="$plugindir/wpps-scripts/"
-fi
-
-#Go to wp-content directory.
-cd "$wpcontentdir";
-sh "${scriptsdir}install-script-dependencies.sh" -c $cwdiswppslinter
+# Run setup.
+source ./setup.sh
 
 # Run the lint command from the wp-content directory.
 if [ "$fix" = "1" ]; then
-	npm run lint:js "$plugindir" -- --config "$scriptsdir.eslintrc" --fix;
+	npm run lint:js "$plugindir" -- --config .eslintrc --fix;
```

Comment:
That's great to not have to have `"$scriptsdir.eslintrc"`

Reply:
Something must be wrong with my syntax here. [CCI doesn't seem to be using `.eslint`](https://app.circleci.com/pipelines/github/studiopress/fse-studio/510/workflows/b235003c-dd51-488d-8d5b-4a0d21b28f8e/jobs/1067?invite=true#step-103-12)